### ISSUE TITLE
Reword ipykernel installation message 

### ISSUE THREAD

**Issue by greazer** [microsoft/vscode-jupyter#9159]
Even people who are familiar with Jupyter may not know why "ipykernel" needs to be installed sometimes. We had an internal case-in-point.

The message we show to the user should provide some level of explanation. Something like "Either the ipykernel or jupyter package needs to be installed in the current environment to support running cells in [interactive-window | jupyter notebooks]. Would you like us to install ipykernel for you?" (Learn more...)

Learn more could discuss the difference.

**Comment by rchiodo** [comment]
Here's the last discussion when we switched to a modal dialog:

https://github.com/microsoft/vscode-jupyter/issues/5798

**Comment by DonJayamanne** [comment]
I agree with @rchiodo . Then again, I guess the suggestion for the change is because @greazer  feels the current message is not clear yet again. But does this mean we'll stop re-visiting this message? I think we've changed this particular message more than 3 times already. Not that its a bad thing, but feel perhaps we're beating a dead horse and its possible its just complicated and we need to make it very simple (& i thought thats why we ended up with the message we have today).

Also note: `ipykernel or jupyter` isn't correct, as installing jupyter doesn't install `ipykernel`

**Comment by greazer** [comment]
Seems that the wording message may not be necessary (other than possibly mentioning "jupyter dependency") but at least having a link to learn more about why it's needed would be good.
